grisp/default/common/build/nifs/grisp_rtems_nif.c

```diff
@@ -95,10 +98,32 @@ static ERL_NIF_TERM clock_set(ErlNifEnv *env, int argc,
   }
 }
 
+static ERL_NIF_TERM unmount_nif(ErlNifEnv *env, int argc,
+                                const ERL_NIF_TERM argv[]) {
+  char path[0xFF];
+  int len = 0;
+  int status;
+
+  if ((len = enif_get_string(env, argv[0], path, sizeof(path),
+                             ERL_NIF_LATIN1)) <= 0)
+    return enif_make_badarg(env);
+
+  status = unmount(path);
+
+  if (status < 0)
+    return enif_make_tuple2(
+        env, enif_make_atom(env, "error"),
+        enif_make_string(env, strerror(errno), ERL_NIF_LATIN1));
+
+  return enif_make_atom(env, "ok");
+}
+
 static ErlNifFunc nif_funcs[] = {
     {"clock_get_ticks_per_second", 0, clock_get_ticks_per_second},
     {"clock_get_ticks_since_boot", 0, clock_get_ticks_since_boot},
     {"clock_get_tod_nif", 0, clock_get_tod},
-    {"clock_set_nif", 1, clock_set}};
+    {"clock_set_nif", 1, clock_set},
+    {"unmount_nif", 1, unmount},
```

Comment:
This should be `{"unmount_nif", 1, unmount_nif}` or the function needs to be named unmount.